extensions/wikia/ThemeDesigner/UploadWordmarkFromFile.class.php

```diff
@@ -1,59 +1,60 @@
 <?php
+
 /*
  * @author Federico "Lox" Lucignano
  */
 
 class UploadWordmarkFromFile extends UploadFromFile {
 	const FILEDIMENSIONS_ERROR = 30;
 	const FILETYPE_ERROR = 31;
-	
-	public function verifyUpload(){
+
+	public function verifyUpload() {
 		$this->getTitle(); //will fill in final destination and extension
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)